frontend/wasted-app/package.json

```diff
@@ -9,6 +9,8 @@
   },
   "dependencies": {
     "@expo/vector-icons": "^12.0.5",
+    "@react-native-community/datetimepicker": "^3.5.2",
+    "@react-native-picker/picker": "^2.2.0",
```

Comment:
I think Wix's RN UI Lib already had datetimepicker and picker components, no?

Reply:
Yes, I'm using Wix library. It uses native libraries so I need to import those dependencies as the documentation explains
<img width="430" alt="Captura de pantalla 2021-11-15 a las 15 08 17" src="https://user-images.githubusercontent.com/51116828/141787193-3c254473-c45b-43ec-9c6a-89ca03fbbe74.png">

Reply:
ah gotcha, all good